data/campaigns/Winds_of_Fate/scenarios/10_Fire_Meets_Steel.cfg

```diff
@@ -319,7 +308,7 @@
                 carryover_percentage=0
             [/gold_carryover]
             [note]
-                description= _ "The new eyrie quadruples village income and trains veteran recruits"
+                description= _ "The new eyrie doubles village output and trains veteran recruits"
```

Comment:
How about using "income and support" to be clearer?

Reply:
Good idea. Applied the change.